torch/_inductor/cudagraph_trees.py

```diff
@@ -555,11 +571,7 @@ def __init__(
         # of CUDAGraphNodes. We should not be freeing parameters, not do we need to account for
         # their liveness (they are static), so we need to compute which outputs are aliases of
         # parameters
-        self.static_input_storage_ptrs: Set[int] = {
-            inputs[i].untyped_storage().data_ptr()
-            for i in self.wrapped_function.static_input_idxs
-        }
-        self.output_is_alias_of_static_inputs: OutputList[int] = []
+        self.output_is_alias_of_persistent_static_inputs: OutputList[int] = []
```

Comment:
It seems worth commenting the difference between "persistent static" and "static"? Esp. since we had to talk about this in 1:1 to clarify it?

Reply:
It's commented below, but i'll move the comment here and link it below